lib/puppet/ssl/host.rb

```diff
@@ -484,6 +478,30 @@ def load_certificate_bundle(bundle_string)
     end
   end
 
+  # Fetches the crl bundle from the CA server; because this is used for the first
+  # fetch of the crl, we need to pass in a certificate store that has no crls
+  # attached to it, but still utilizes the localcacert.
```

Comment:
Some kind of comment regarding the effects of `get_crls` raising during a `replace_file`, either in the description or code would be nice.